org.jacoco.doc/docroot/index.html

```diff
@@ -68,6 +68,12 @@
       <td>Ant <i>(all other dependencies included)</i></td>
     </tr>
     <tr>
+      <td><span class="el_jar">jacococli.jar</span></td>
+      <td>no</td>
+      <td>JaCoCo Command Line Interface</td>
```

Comment:
@marchof at the bottom of this page we refer to `repo.html` that lists all deployed artifacts and their coordinates, this artifact will be deployed, but `repo.html` wasn't updated to mention its coordinates.